datarockets-style.gemspec

```diff
@@ -30,6 +30,9 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_dependency "pry-byebug"
+  spec.add_dependency "rake", "~> 13.0"
+  spec.add_dependency "rspec", "~> 3.10"
```

Comment:
it should be development dependency. I don't want to have it as a dependency in the project

Reply:
@Roma fixed

Reply:
@roman-dubrovsky fixed